oauth2_provider/models.py

```diff
@@ -449,7 +449,7 @@ class AbstractIDToken(models.Model):
         settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=True, null=True,
         related_name="%(app_label)s_%(class)s"
     )
-    token = models.TextField(unique=True)
+    token = models.TextField()
```

Comment:
or should we move to uuid?

Reply:
but isn't this value _not_ actually a uuid, but a raw JWT (see the `get_claims` method on this class?), in which case a TextField is the appropriate type?

Reply:
this field needs to be a text field because that will be persisted a long string (the jwt token) on this, but, I think that independently of the database backend, it seems there is no reason for this field to be a unique field.

Reply:
but, in terms of security, I think that this token really needs to be unique. so every request will receive a valid response with a different valid token. in the token generation process, more information can be put to guarantee that uniqueness won't be invalid. I don't know, but I think that need another approach to resolve this problem for mysql backends

Reply:
An idea would be to add an additional unique CharField that contains a hash of the token. So we can make sure it is really unique on an mysql backend.